arxiv/urls/__init__.py

```diff
@@ -0,0 +1,12 @@
+from ..identifier import Identifier
+
+def pdf_canonical_url (identifier: Identifier) -> str:
+    # This won't include the version unless identifier 
+    # was explicitly initialized with a version
+    return f'https://arxiv.org/pdf/{identifier.idv}'
```

Comment:
Not sure where this is used but might be good to use config.CANONICAL_SERVER

But maybe some other time.